docs/usage/PRODUCT_SYNC.md

```diff
@@ -146,7 +140,7 @@ More examples of how to use the sync
 A utility method provided by the library to compare a Product with a new ProductDraft and results in a list of product
  update actions. 
 ```java
-List<UpdateAction<Product>> updateActions = ProductSyncUtils.buildActions(product, productDraft, productSyncOptions);
+List<UpdateAction<Product>> updateActions = ProductSyncUtils.buildActions(product, productDraft, productSyncOptions, attributesMetaData);
```

Comment:
Why don't you use clear naming conventions for variables? It should be always clear what is old/existing and what is new. For example: `ProductSyncUtils.buildActions(oldProduct, newProductDraft, productSyncOptions, attributesMetaData);` and would follow this convention everywhere.